examples/classification/README.md

```diff
@@ -34,3 +38,85 @@ python classification.py mobilenet_v2 --batch_size 8 --num_cores 4
+batch_loader = _BatchLoader(data=load_data(resnet_stub), batch_size=1, iterations=1)
+
+for batch in batch_loader:
+    out = client.classify(images=batch, )
+    print(f"outputs:{out}")
+```
+
+### SparseZoo Stubs
+[SparseZoo](http://sparsezoo.neuralmagic.com/) is a constantly-growing repository of sparsified (pruned and 
+pruned-quantized) models with matching sparsification recipes for neural networks. It simplifies and accelerates your time-to-value in building performant deep learning models with a collection of inference-optimized models and recipes to prototype from.
+
+Available via API and hosted in the cloud, the [SparseZoo](http://sparsezoo.neuralmagic.com/) contains both 
+baseline and models sparsified to different degrees of inference performance vs. baseline loss recovery. Recipe-driven approaches built around sparsification algorithms allow you to take the models as given, transfer-learn from the models onto private datasets, or transfer the recipes to your architectures.
+
+Each model in the SparseZoo has a specific stub that identifies it. The stubs are made up of the following structure:
+
+`DOMAIN/SUB_DOMAIN/ARCHITECTURE{-SUB_ARCHITECTURE}/FRAMEWORK/REPO/DATASET{-TRAINING_SCHEME}/SPARSE_NAME-SPARSE_CATEGORY-{SPARSE_TARGET}`
+
+Learn more at 
+- [SparseZoo Docs](https://docs.neuralmagic.com/sparsezoo/)
+- [SparseZoo Website](https://sparsezoo.neuralmagic.com/) 
+- [SparseZoo GitHub Repository](https://github.com/neuralmagic/sparsezoo)
+
+A few classification models from [SparseZoo](https://sparsezoo.neuralmagic.com/):
+
+| Model Name     |      Stub      | Description |
+|----------|-------------|-------------|
+| resnet-pruned-moderate | zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate |This model is a sparse, [ResNet-50](https://arxiv.org/abs/1512.03385) model that achieves 99% of the accuracy of the original baseline model (76.1% top1). Pruned layers achieve 88% sparsity.|
+|resnet-pruned-conservative|zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-conservative|This model is a sparse, [ResNet-50](https://arxiv.org/abs/1512.03385) model that achieves full recovery original baseline model accuracy (76.1% top1). Pruned layers achieve 80% sparsity.|
+|resnet-pruned-quantized-moderate|zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned_quant-moderate|This model is a sparse, INT8 quantized ResNet-50 model that achieves 99% of the original baseline accuracy. The average model sparsity is about 72% with pruned layers achieving between 70-80% sparsity. This pruned quantized model achieves 75.46% top1 accuracy on the ImageNet dataset.|
+|mobilenetv1-pruned-quantized|zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned_quant-moderate|Pruned [MobileNet](https://arxiv.org/abs/1704.04861) architecture with width=1.0 trained on the ImageNet dataset. Recalibrated performance model: within 99% of baseline validation top1 accuracy (70.9%)|
```

Comment:
let's expand out a bit more to include mobilenet and resnet baseline models so users can quickly compare dense to pruned and pruned quantized. Additionally, let's include the pruned models for mobilenet